### PR TITLE
Add help tooltip for search syntax

### PR DESCRIPTION
**Pull request by Zangr**
Add help tooltip for search syntax.
Note: this PR adds [ant design](https://ant.design/) as a new component library. 

search-tooltip

Update: 04-18-2019
Screen Shot 2019-04-18 at 2 28 10 PM

Screen Shot 2019-04-18 at 2 23 20 PM

### CODE REVIEW

File: mlflow/server/js/src/components/ExperimentView.js [hunk 1]
```diff
@@ -265,6 +266,18 @@ class ExperimentView extends Component {
     const compareDisabled = Object.keys(this.state.runsSelected).length < 2;
     const deleteDisabled = Object.keys(this.state.runsSelected).length < 1;
     const restoreDisabled = Object.keys(this.state.runsSelected).length < 1;
+    const searchInputHelpTooltipContent = (
+      <div className="search-input-tooltip-content">
+        Click here to learn more about the search syntax.<br/>
+        <a
+          href="https://mlflow.org/docs/latest/tracking.html"
```
Comment (aarondav) [inline mlflow/server/js/src/components/ExperimentView.js]: Btw, importantly, do we currently document our search syntax @mparkhe? Do we have a place that we plan to document it?

Comment (mateiz) [reply mlflow/server/js/src/components/ExperimentView.js]: Yeah, we should link to a location within that page eventually if possible.

Comment (mparkhe) [reply mlflow/server/js/src/components/ExperimentView.js]: 👍 
cc: @dbczumar

File: mlflow/server/js/src/components/ExperimentView.js [hunk 2]
```diff
@@ -265,6 +266,18 @@ class ExperimentView extends Component {
...
     const restoreDisabled = Object.keys(this.state.runsSelected).length < 1;
+    const searchInputHelpTooltipContent = (
+      <div className="search-input-tooltip-content">
+        Click here to learn more about the search syntax.<br/>
+        <a
+          href="https://www.mlflow.org/docs/latest/search-syntax.html"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn more
```
Comment (stbof) [inline mlflow/server/js/src/components/ExperimentView.js]: Replace 

Click here to learn more about the search syntax.

with 

Search runs within a single experiment or a group of experiments using a simplified version of the SQL ``WHERE`` clause.

It's then obvious that you click the **Learn more** link following the sentence to learn about the syntax.

**Comment by stbof** [comment]
LGTM.

**Review by smurching — APPROVED** [review]
LGTM as well, thanks @Zangr

**Pull request merged**